### PR TITLE
Chat Module

### PR DESCRIPTION
**Pull request by nmalzieu**

### CODE REVIEW

File: src/browser/chat/ChatRoomPage.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    {sender}
+    : {content}
+  </View>
+);
+
+class ChatRoomMessages extends Component {
+  componentWillMount() {
+    this.props.loadMessages(this.props.chatRoom.id);
+  }
+  onInputKeyDown = (event) => {
```
Comment (e-founders) [inline src/browser/chat/ChatRoomPage.js:19]: Prefer using an input type="submit" in the form to automatically the submit event when pressing Enter (this is better for accessibility and more standard overall).

File: src/browser/chat/ChatRoomPage.js [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    const kickUser = (userId) => () => {
+      this.props.kickUserFromRoom(userId, chatRoomId);
+    };
+
+    if (chatRoom) {
+      const chatRoomUsers = chatRoom.presence
+        ? chatRoom.presence
+        : {};
+
+      let chatRoomUsersList = [];
```
Comment (e-founders) [inline src/browser/chat/ChatRoomPage.js:74]: Accumulate your list items in an array directly using chatRoomUsers.map (style)

File: src/browser/chat/ChatRoomPage.js [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+class ChatRoomPage extends Component {
+  componentDidMount() {
+    if (this.props.viewer) {
+      this.props.joinRoom(this.props.viewer, this.props.params.chatRoomId);
+    }
+  }
+  render() {
+    const chatRoomId = this.props.params.chatRoomId;
+    let chatRoom = null;
+    this.props.chatRooms.forEach(theChatRoom => {
```
Comment (e-founders) [inline src/browser/chat/ChatRoomPage.js:59]: Use Array.find to get the proper chatRoom using a one-liner :
`const chatRoom = this.props.chatRooms.find((cr) => cr.id === chatRoomId);`

File: src/browser/chat/ChatRoomPage.js (hunk 1 above)
Comment (nmalzieu) [reply src/browser/chat/ChatRoomPage.js:19]: Ok ! I just saw that it was done that way here : https://github.com/nmalzieu/este-chat/blob/master/src/browser/todos/NewTodo.js#L12 😄

File: src/browser/chat/ChatRoomPage.js (hunk 2 above)
Comment (nmalzieu) [reply src/browser/chat/ChatRoomPage.js:74]: Ok, I wanted to handle my if..else but didn't know I could do it directly in `.map`

File: src/browser/chat/ChatRoomPage.js (hunk 3 above)
Comment (nmalzieu) [reply src/browser/chat/ChatRoomPage.js:59]: Indeed, a lot cleaner 👍

**Pull request closed without merging**